packages/block-editor/src/components/iframe/index.js

```diff
@@ -103,14 +103,13 @@ function Iframe( {
 	forwardedRef: ref,
 	...props
 } ) {
-	const { resolvedAssets, isPreviewMode } = useSelect( ( select ) => {
+	const { isPreviewMode, url } = useSelect( ( select ) => {
 		const settings = select( blockEditorStore ).getSettings();
 		return {
-			resolvedAssets: settings.__unstableResolvedAssets,
```

Comment:
Perhaps we should leave the existing setting alone for now and use it in the fallback.

Reply:
In the future I could even see the whole document HTML being a setting that can be used for the fallback option